src/Command/ImportScorerCommand.php

```diff
@@ -0,0 +1,184 @@
+    }
+
+    /**
+     * import scorer from data file to the database
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->stopwatch->start('import');
+        $this->symfonyStyle->title('Human-Scoring Import');
+        if (!$input->hasParameterOption(self::OPTION_OVERWRITE)) {
+            $output->writeln(['if you want overwrite scorer data in database use flag \'--overwrite\'', '']);
+        }
+        try {
+            $data = $this->dataProvider->getIterator($this->path);
+            $this->persist($data);
+            $this->flush();
+        } catch (\Exception $e) {
+            $this->symfonyStyle->warning($e->getMessage());
+            return 1;
+        }
+            $event = $this->stopwatch->stop('import');
+            $this->showStopwatchData($event->getMemory(),$event->getDuration ());
+            return 0;
+    }
+
+
+    /**
+     * if there is option '--overwrite' persist
+     * persist scorers
+     *
+     * @param $data
+     */
+    private function persist($data)
+    {
+        foreach ($data as $scorer) {
+            $dbScorer = $this->objectManager->getRepository(Scorer::class)->findOneBy(['username' => $scorer['username']]);
+            if (!$this->input->hasParameterOption(self::OPTION_OVERWRITE)) {
+                if (!($dbScorer instanceOf Scorer)) {
+                    $this->addNewScorer($scorer);
+                }
+            }
+            if ($dbScorer instanceOf Scorer) {
+                $this->updateScorer($dbScorer, $scorer);
+            }
+        }
+    }
+
+    /**
+     * if there is '--force' option, flush the data in the database
+     */
+    private function flush()
+    {
+        if ($this->input->hasParameterOption(self::OPTION_FORCE)) {
+            $this->objectManager->flush();
+            $this->symfonyStyle->success('import done successfully !!');
+        } else {
+            $this->symfonyStyle->block('try import done successfully !!', 'info', 'fg=black;bg=blue', ' ', true);
+        }
+    }
+
+    /**
+     * persist a new scorer
+     *
+     * @param $scorer
+     */
+    private function addNewScorer($scorer)
+    {
+        $scorerEntity = new Scorer();
+        $scorerEntity->setUsername($scorer['username']);
+        $scorerEntity->setPassword($this->encoder->encodePassword($scorerEntity, $scorer['password']));
+        $this->objectManager->persist($scorerEntity);
+    }
+
+    private function updateScorer($scorerEntity, $scorer)
+    {
+        $scorerEntity->setPassword($this->encoder->encodePassword($scorerEntity, $scorer['password']));
+        $this->objectManager->persist($scorerEntity);
+    }
+
+    /**
+     * convert and show StopWatch data
+     *
+     * @param $memory
+     * @param $duration
+     */
+    private function showStopwatchData($memory, $duration)
```

Comment:
Probably this function could be extracted into a trait so later you can reuse it in other commands also like we did here: https://github.com/oat-sa/depp-teacher-portal/blob/master/src/Command/StopWatchTrait.php